### PR TITLE
python server fixes

### PR DESCRIPTION
**Pull request by aviemzur**

### CODE REVIEW

**Comment by aviemzur** [comment]
R: @assapin

**Review by assapin — APPROVED** [review]
worth looking into Flask blueprints as an alternative, more native mechanism for creating this delegator pattern.

File: liminal/core/util/class_util.py [hunk 1]
```diff
@@ -46,20 +46,26 @@ def find_subclasses_in_packages(packages, parent_class):
...
     """
     if isinstance(package, str):
         package = importlib.import_module(package)
     results = {}
     for loader, name, is_pkg in pkgutil.walk_packages(package.__path__):
         full_name = package.__name__ + '.' + name
-        results[full_name] = importlib.import_module(full_name)
-        if recrsive and is_pkg:
-            results.update(import_module(full_name))
+        if not name == 'liminal_python_server':
```
Comment (assapin) [inline liminal/core/util/class_util.py:65]: why is this line needed?
